File: samples/CslaLight/vb/Rolodex/Business/Server/Data/DataConnection.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Configuration

Namespace Rolodex.Business.Data
  Public NotInheritable Class DataConnection
	Private Sub New()
	End Sub
	Public Shared ReadOnly Property ConnectionString() As String
	  Get
		Return System.Configuration.ConfigurationManager.ConnectionStrings("WcfHostWeb.Properties.Settings.RollodexConnectionString").ConnectionString
	  End Get
	End Property
  End Class
End Namespace

File: Samples/CslaLight/cs/Rolodex/Business/Server/Data/DataConnection.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Configuration;

namespace Rolodex.Business.Data
{
  public static class DataConnection
  {
    public static string ConnectionString
    {
      get
      {
        return System.Configuration.ConfigurationManager.ConnectionStrings["WcfHostWeb.Properties.Settings.RollodexConnectionString"].ConnectionString;
      }
    }
  }
}
